xtext-website/documentation/310_eclipse_support.md

```diff
@@ -604,6 +613,36 @@ public class MyDslUiModule extends AbstractMyDslUiModule {
 }
 ```
 
+## Mark Occurrences {#maroccurrences}
+Xtext-based editors are able to highlight all occurrences of a certain element in the opened DSL file. Once the user selects an element via double click while the `Toggle Mark Occurrences` button is enabled, all occurrences are highlighted with corresponding markers on the right side of the editor.
```

Comment:
double click is new to me. thought a single one is sufficient

Reply:
Fixed.